File: langs/Cowsay.java
public class Cowsay{
    public static void main(String[] a){
        String m=a.length>0?String.join(" ",a):"Hello, World!";
        System.out.print(" "+"_".repeat(m.length()+2)+"\n< "+m+" >\n "+"-".repeat(m.length()+2)+"\n        \\   ^__^\n         \\  (oo)\\_______\n            (__)\\       )\\/\\\n                ||----w |\n                ||     ||\n");
    }
}

File: langs/cowsay.go
package main

import (
	"fmt"
	"os"
	"strings"
)

func main() {
	m := "Hello, World!"
	if len(os.Args) > 1 {
		m = strings.Join(os.Args[1:], " ")
	}
	fmt.Printf(" %s\n< %s >\n %s\n        \\   ^__^\n         \\  (oo)\\_______\n            (__)\\       )\\/\\\n                ||----w |\n                ||     ||\n",
		strings.Repeat("_", len(m)+2), m, strings.Repeat("-", len(m)+2))
}
